src/agent/onefuzz-supervisor/src/main.rs

```diff
@@ -84,12 +90,58 @@ fn version() {
 }
 
 fn licenses() -> Result<()> {
-    use std::io::{self, Write};
     io::stdout().write_all(include_bytes!("../../data/licenses.json"))?;
     Ok(())
 }
 
+fn redirect(opt: RunOpt) -> Result<()> {
+    let log_path = opt
+        .redirect_output
+        .expect("redirect should only be called with log_path");
```

Comment:
We can just pass the redirect output as a parameter and avoid the check here.

Reply:
Given we need to pull out other arguments out of `opt`, that felt like it reduced clarity.